# Information of the place holders

# Number of place holders
int64 num_place_holders

# Info 1=door
int64[] is_a_door

# Position of place holders
geometry_msgs/Pose[] pose

# Area if
int64[] area_id
